src/scaffoldmaker/meshtypes/meshtype_3d_ostium1.py

```diff
@@ -725,8 +725,9 @@ def generateOstiumMesh(region, options, trackSurface, centrePosition, axis1, sta
     for v in range(vesselsCount):
         if vesselMeshGroups or ostiumMeshGroups:
             rowMeshGroups = []
-            for i in range(elementsCountAlong):
+            for i in range(elementsCountAlong - 1):
                 rowMeshGroups.append(copy.copy(vesselMeshGroups[v]) if vesselMeshGroups else [])
+            rowMeshGroups += [ostiumMeshGroups]
```

Comment:
This will have broken the heart scaffold for human and rat which use common ostium.
I think it is right to include the last bit as being both bladder + neck AND ureter; the reason is that our primary use is to project data onto, and some of the data for ureter could/should project onto these elements.

Reply:
I think it's better to put the last layer of the ostium mesh only in one meshGroup. Since the ureters enter the bladder and make a slit on its surface. So, the elements around the ostiums are only part of the ureter and not the bladder. So, I want to set the 
 "vesselMeshGroups=[[ureterMeshGroup]]" and the "ostiumMeshGroups=[]"

Reply:
We make anatomical groups to support fitting and registration and for that purpose, the elements in the inlet zone are as much ureter as bladder. If, for example, we had data points for the outside of the ureter, some of them would need to project onto those elements, and likewise for the bladder.
I'm more concerned about there being a second row of elements that are only ureter, so the bladder has parts that are not bladder... Ideally I'd have just one element along the ostium to reduce mesh complexity, but I admit it's hard to get around the corner without distortions. I'm reasonably happy with 1 element along using:
Ostium diameter 2.2
Ostium length 0.5
Vessel end length factor 2.0
I notice the ostium has a problem with multiple elements along over such a small length due to something I've done to limit wall thicknesses, but if we decide we need 2 elements to go around this curve we'd want both to be in both bladder and ureter groups. BTW do you have left and right ureter groups?
The changes to the ostium code do break the heart scaffold & need to be undone.